src/express/express.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require(`express`);
+const app = express();
+const APPLICATION_PORT = 8080;
```

Comment:
пытаемся вычитать порт из переменной среды или 8080 как дефолтный
```suggestion
const APPLICATION_PORT = process.env.PORT || 8080;
```

Reply:
Согласен, сделаю, спасибо